Source/Core/VideoCommon/VertexShaderGen.cpp

```diff
@@ -318,10 +314,8 @@ ShaderCode GenerateVertexShaderCode(API_TYPE api_type, const vertex_shader_uid_d
       }
       else
       {
-        // The following assert was triggered in House of the Dead Overkill and Star Wars Rogue
-        // Squadron 2
-        //_assert_(0); // should have normals
-        out.Write("o.tex%d.xyz = o.tex%d.xyz;\n", i, texinfo.embosssourceshift);
+        // Even if inputform ABC1 is set, it only uses AB11
```

Comment:
It might be nice to note known games which do this, since it's rare.  (Did anyone ever test "House of the Dead Overkill"?)